topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,383 @@
+---
+layout: tutorial_hands_on
+
+title: Data submission using ENA upload Tool
+questions:
+- How to prepare sequences for submission to ENA?
+- How to upload raw sequences to ENA?
+objectives:
+- Manage sequencing files (ab1, FASTQ, FASTA, FASTQ.GZ)
+- Clean sequences in an automated and reproducible manner
+- Perform alignments for each sequence
+- Have the necessary sequence format to submit to ENA
+- Submit raw reads to ENA using the ENA upload Tool
+time_estimation: 2h
+key_points:
+- Clean raw ab1 sequences and compare filtered sequences to NCBI nucleotidic database
+- Submit cleaned and unique sequences to European Nucleotide Archive (ENA) resource
+contributions:
+    authorship:
+        - Najatamk
+        - yvanlebras
+    testing:
+        - PaulineSGN
+        - yvanlebras
+    funding:
+        - pndb
+
+subtopic: ecologymetadatamgt
+---
+
+
+This tutorial will guide you through the necessary steps to manage and prepare sequencing files (ab1, FASTQ, FASTA) for submission to the genomic database ENA.
+This workflow will take you from raw sequences in AB1 format through all the necessary steps to integrate these sequences into the ENA genomic database. We will convert the files into FASTQ and FASTA formats after performing quality control.
+Additionally, we will perform alignments with the NCBI database to ensure the accuracy of your sequences.You will then need to fill a metadata Excel template to use the ENA upload Tool.
+The worklow is made of 17 Galaxy tools, we will  present them and explain what they do.
+The goal is to present an accessible and reproductible workflow for data submission.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Prepare raw data
+
+> <hands-on-title> Data Upload </hands-on-title>
+> 1. **Create a new history** for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. **Import** the raw sequences files.
+>
+>    ```
+>    https://data.indores.fr/api/access/datafile/3673
+>    https://data.indores.fr/api/access/datafile/3609

```

Comment:
sorry, didnt notice this before, but could you put the data on Zenodo?

(Then you can already name the files like `A2_RC_8F2_B.pl_HCOI.ab1` and save that step)

Reply:
These data are already on a trusted french data wharehouse, indores, similar to Zenodo. I prefer not dupliating files. Is it ok for you ?

Reply:
Could you put it also in Zenodo, and put the zenodo link in the metadata. You can then use the indores URLs here, but then Galaxy admins can still choose to create data libraries with all data for tutorials (useful for e.g. trainings in places with bad connections)